frameworks/hdfs/tools/docker-client/Dockerfile

```diff
@@ -1,37 +1,39 @@
 FROM ubuntu:16.04
 
-RUN apt-get -y update && \
-    apt-get -y upgrade && \
-    apt-get -y install wget software-properties-common
-
-RUN wget https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
-    tar xf hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
-    rm hadoop-2.6.0-cdh5.9.1-dcos.tar.gz
-
-RUN apt-get update && \
-    apt-get install -y default-jre && \
-    apt-get install -y \
-    curl \
-    dnsutils \
-    less \
-    vim
-
 ENV HADOOP_VERSION hadoop-2.6.0-cdh5.9.1
 ENV HDFS_SERVICE_NAME hdfs
 ENV JAVA_HOME /usr/lib/jvm/default-java
 
+RUN apt-get -y update && \
+    apt-get -y upgrade && \
+    apt-get -y install \
+      curl \
+      default-jre \
+      dnsutils \
+      less \
+      software-properties-common \
+      vim \
+      wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN wget https://downloads.mesosphere.com/hdfs/assets/$HADOOP_VERSION-dcos.tar.gz && \
+    tar xf $HADOOP_VERSION-dcos.tar.gz && \
+    rm $HADOOP_VERSION-dcos.tar.gz
+
+
 # the env var is required for a non-interactive install of krb5-user
-RUN export DEBIAN_FRONTEND=noninteractive && apt-get install -y krb5-user
-COPY krb5.conf /etc/krb5.conf
-RUN chown -R nobody /etc
+RUN export DEBIAN_FRONTEND=noninteractive && \
```

Comment:
Any reason to not also include `krb5-user` in the `apt-step` above?

Reply:
Ah so I tried that the first time and it didn't work for some reason. Tried it again and seems to be fine. Will just include it above.

Reply:
You don't need to export it as it is only required for the `apt-get install` command. For example from https://github.com/elezar/tf_workspace/blob/master/Dockerfile.cpu#L4:
```
RUN apt-get update && \
    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
        git \
        tree \
        wget \
        zsh \
    && \
```